docs/agent/hostname_force_config_as_canonical.md

```diff
@@ -0,0 +1,8 @@
+Agent hostname config as canonical 
+
+If `hostname` is set in `datadog.yaml` and the value starts with `ip-` or `domu`, this hostname is not used as a canonical hostname.
+More information about what a canonical hostname is can be found at [How does Datadog determine the Agent hostname?](https://docs.datadoghq.com/agent/faq/how-datadog-agent-determines-the-hostname/?tab=agentv6v7#agent-versions).
+
+To use an hostname starting with `ip-` or `domu` as a canonical hostname, set the config `hostname_force_config_as_canonical` at `true` in datadog.yaml.
+It creates a new canonical hostname which receives new metrics. The old canonical hostname contains old metrics. 
```

Comment:
I would put these 2 sentences in a new section called "Impact of enabling `hostname_force_config_as_canonical`" or something similar.

Reply:
are you sure this is what happens in practice btw?

IIRC the behavior if the option is enabled on an existing Agent is that the hostname will remain the same in-app, until the support team runs the command that clears the existing host aliases

Reply:
I have checked it again and you are right, I confuse with another use case.